src/organisms/footer/Footer.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import  Copyright from "../../atoms/copyright";
+import  DownloadOptions  from "../../molecules/downloadOptions";
+import "./style.css";
+
+function footer() {
+    return (
+        <div className="footer display-flex">
+            <div className="footer__left display-flex">
+                <Copyright />
+                <DownloadOptions />
+            </div>
+            <div className="footer__right">
+                <button className="footer__facebook">
+                    <i className="fa fa-facebook" aria-hidden="true"></i>
+                </button>
+                <button className="footer__twitter">
+                    <i className="fa fa-twitter" aria-hidden="true"></i>
+                </button>
+                <button className="footer__instagram">
+                    <i className="fa fa-instagram" aria-hidden="true"></i>
+                </button>
+                <button className="footer__linkedin">
```

Comment:
`<button className="footer__instagram">
       <i className="fa fa-instagram" aria-hidden="true"></i>
  </button>`
this code is repeated, only icon classname is different. You can create common component for this and send icon as prop